### PR TITLE
Metadata: isaaclevin.presencelight version 1.0.171

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21819)

### CODE REVIEW

File: manifests/i/isaaclevin/presencelight/1.0.171/isaaclevin.presencelight.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: isaaclevin.presencelight
+PackageVersion: 1.0.171
```
Comment (OfficialEsco) [inline manifests/i/isaaclevin/presencelight/1.0.171/isaaclevin.presencelight.installer.yaml:4]:
```suggestion
PackageVersion: 1.0.171.0
```

File: manifests/i/isaaclevin/presencelight/1.0.171/isaaclevin.presencelight.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: appx
+  Scope: machine
+  InstallerUrl: https://github.com/isaacrlevin/PresenceLight/releases/download/v1.0.171/PresenceLight.Package_1.0.171.0_x86.appxbundle
+  InstallerSha256: 7A771B7198A58A450BA65D5A0A4FF3583936243CEEE3038024C678B27AF90357
```
Comment (OfficialEsco) [inline manifests/i/isaaclevin/presencelight/1.0.171/isaaclevin.presencelight.installer.yaml:16]:
```suggestion
  InstallerSha256: 7A771B7198A58A450BA65D5A0A4FF3583936243CEEE3038024C678B27AF90357
  SignatureSha256: 07054CFE946F0C9F80918A6CD901312E25EA7E8E84E04A406559576624E60837
  PackageFamilyName: 37828IsaacLevin.197278F15330A_jvewcxq8vj8qt
```

File: manifests/i/isaaclevin/presencelight/1.0.171/isaaclevin.presencelight.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: isaaclevin.presencelight
+PackageVersion: 1.0.171
```
Comment (OfficialEsco) [inline manifests/i/isaaclevin/presencelight/1.0.171/isaaclevin.presencelight.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 1.0.171.0
```

File: manifests/i/isaaclevin/presencelight/1.0.171/isaaclevin.presencelight.yaml [hunk 4]
```diff
@@ -1,19 +1,8 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
 PackageIdentifier: isaaclevin.presencelight
 PackageVersion: 1.0.171
```
Comment (OfficialEsco) [inline manifests/i/isaaclevin/presencelight/1.0.171/isaaclevin.presencelight.yaml:4]:
```suggestion
PackageVersion: 1.0.171.0
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
If you are going to spam old Manifest at least do it right.

File: manifests/i/isaaclevin/presencelight/1.0.171/isaaclevin.presencelight.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: appx
+  Scope: machine
```
Comment (OfficialEsco) [inline manifests/i/isaaclevin/presencelight/1.0.171/isaaclevin.presencelight.installer.yaml:14]:
```suggestion
MinimumOSVersion: 10.0.17134.0
InstallModes:
- interactive
- silent
Installers:
- InstallerLocale: en-US
  Architecture: x86
  InstallerType: appx
  Scope: user
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Comment by vedantmgoyal2009** [comment]
Where can we find `SignatureSha256`?

**Comment by denelon** [comment]
@vedantmgoyal2009 the SignatureSha256 can be calculated using the Windows Package Manager client.

```
PS C:\Users\denelon> winget hash --help
Windows Package Manager v1.0.11971
Copyright (c) Microsoft Corporation. All rights reserved.

Computes the hash of a local file, appropriate for entry into a manifest.  It can also compute the hash of the signature file of an MSIX package to enable streaming installations.

usage: winget hash [-f] <file> [<options>]

The following arguments are available:
  -f,--file  File to be hashed

The following options are available:
  -m,--msix  Input file will be treated as msix; signature hash will be provided if signed

More help can be found at: https://aka.ms/winget-command-hash
```

**Comment by vedantmgoyal2009** [comment]
Is it also available for appx files?

**Comment by denelon** [comment]
@vedantmgoyal2009 yes it should work for ".msix", ".msixbundle", ".appx", ".appxbundle".

**Comment by ItzLevvie** [comment]
cc @denelon @palenshus: "Azure-Pipeline-Passed" label needs to be added to this PR.

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**